README.md

```diff
@@ -60,3 +60,21 @@ parameters:
                 - 2     # /Home
                 - 43    # /Media
 ```
+
+## Preselected content type
+To set the content type you have to provide 'contentTypeIdentifier' in the config of the 'contentDiscover' event.
+Example:
+```javascript
+/**
+* ...
+* @param config.visibleMethod {String} which tab should be open. Default: 'browse', possible values: 'browse', 'search', 'create'
+* @param config.contentTypeIdentifier {String} content type identifier. Default: none, example values: 'image', 'blog', 'article', 'blog_post'
```

Comment:
Should maybe also document inline:
- That if there is bundle yml config for location as well then first option will be preselected as location.
- That for the time being `contentTypeIdentifier` is limited to Create tab, and config that works across all tabs will be added in the future.  _(implying it might be different)_